api/External.json

```diff
@@ -20,7 +20,7 @@
             "version_added": true
           },
           "ie": {
-            "version_added": null
+            "version_added": false
```

Comment:
Huh, that _is_ pretty weird...  From what I can tell, in IE11, the property is just a blank object, and isn't actually an object of the API (in other words, something that'll screw up feature detection).

Looking at the spec, an `External` API _should_ be exposed on the Window, which it is in Chrome but not in IE, so I'm leaning towards saying this isn't implemented (in fact, with the way `window.external` is, it's _worse_ than not implemented).

Reply:
It might be that this API was exposed in some contexts (extensions or some special kinds of scripts?) and not others, and this weird binding was left exposed to web content as well. Let's go with false.